src/about.js

```diff
@@ -11,7 +11,8 @@ module.exports = projectLocation => {
       version: packageJson.version,
       description: packageJson.description || '<no description>',
       author: packageJson.author || '<no author>',
-      license: packageJson.license || 'AGPL-v3.0'
+      license: packageJson.license || 'AGPL-v3.0',
+      flowEditorDisabled: Boolean(process.env.BOTPRESS_FLOW_EDITOR_DISABLED)
```

Comment:
I don't understand the reason for that setting to be located in the "About" file?

Reply:
These settings are then available on the frontend as info on bot so I thought it makes sense to have it here. Do you think it should be moved elsewhere and fetched separately?

Reply:
OK, yeah so this might not be totally clear, but this file is actually for storing "About this bot" information, i.e. nothing related to features and configuration.

Please have a look at my branch where I did something similar: https://github.com/botpress/botpress/blob/x-cloud/src/server/static.js#L103

Reply:
OK, makes sense.